core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/AbstractHealthCheckedEndpointGroupBuilder.java

```diff
@@ -155,12 +162,53 @@ public AbstractHealthCheckedEndpointGroupBuilder withClientOptions(
         return this;
     }
 
+    /**
+     * Sets the maximum endpoint ratio of target selected candidates.
+     * @see PartialHealthCheckStrategyBuilder#maxEndpointRatio(double)
+     */
+    public AbstractHealthCheckedEndpointGroupBuilder maxEndpointRatio(double maxEndpointRatio) {
+        if (partialHealthCheckStrategyBuilder == null) {
+            partialHealthCheckStrategyBuilder = new PartialHealthCheckStrategyBuilder();
+        }
+
+        partialHealthCheckStrategyBuilder.maxEndpointRatio(maxEndpointRatio);
+        return this;
+    }
+
+    /**
+     * Sets the maximum endpoint count of target selected candidates.
+     * @see PartialHealthCheckStrategyBuilder#maxEndpointCount(int)
+     */
+    public AbstractHealthCheckedEndpointGroupBuilder maxEndpointCount(int maxEndpointCount) {
```

Comment:
Ditto - How about validating `maxEndpointCount` before instantiating the builder?

Reply:
fixed

Reply:
fixed